pkg/controller/volume/cache/desired_state_of_world_test.go

```diff
@@ -194,6 +194,10 @@ func Test_AddPod_Positive_NewPodNodeExistsVolumeExists(t *testing.T) {
 		t.Fatalf("len(volumesToAttach) Expected: <1> Actual: <%v>", len(volumesToAttach))
 	}
 
+	podScheduled := dsw.GetPods()
+	if len(podScheduled) != 2 {
+		t.Fatalf("len(podScheduled) Expected: <1> Actual: <%v>", len(podScheduled))
```

Comment:
So, the t.Fatalf message should be `"len(podScheduled) Expected: <2> Actual: <%v>"`, not `"len(podScheduled) Expected: <1> Actual: <%v>"`